examples/chart/teleport-cluster/charts/teleport-operator/templates/_helpers.tpl

```diff
@@ -75,3 +76,56 @@ teleport.dev/majorVersion: '{{ include "teleport-cluster.majorVersion" . }}'
 {{- /* This template is a placeholder.
 If we are imported by the main chart "teleport-cluster" it is overridden*/ -}}
 {{- define "teleport-cluster.auth.serviceFQDN" -}}{{- end }}
+
+{{- /* This templates returns "true" or "false"  describing if the CRDs should be deployed.
+If we have an explicit requirement ("always" or "never") things are easy.
+If we don't we check if the operator is enabled.
+However, we cannot just trash the CRDs if the operator is disabled, this causes
+a mass CR deletion and users will shoot themselves in the foot whith this
+(temporarily disabling the operator would cause havoc).
+So we check if there's a CRD already deployed, it that's the case, we keep the CRDs.
+*/ -}}
+{{- define "teleport-cluster.operator.shouldInstallCRDs" -}}
+  {{- if eq .Values.installCRDs "always" -}}
```

Comment:
we need to check `.Values.installCRDs=true|false` to keep compatibility with previous hardcoded values.

Reply:
We're in `operator`, and `.Values.operator.installCRDs` never existed, only `.Values.installCRDs` existed. We don't have access to `.Values.installCRDs`.